### PR TITLE
creates section 3.3.1 for the content directory

### PR DESCRIPTION
**Pull request by rosy1280**
per discussion in OCFL [2019.12.03 Editors meeting](https://github.com/OCFL/spec/wiki/2019.12.03-Editors-Meeting)

c.f. #407

### CODE REVIEW

File: draft/spec/index.html [hunk 1]
```diff
@@ -453,32 +453,34 @@ <h2>Version Directories</h2>
...
-            (including any validators) MUST ignore all directories in the object version directory except for the
-            designated content directory.
-        </p>
-        <p>
-            Every file within a version's content directory MUST be referenced in the <a href="#manifest">manifest</a>
-            section of the inventory. There MUST NOT be empty directories within a version's content directory. A
-            directory that would otherwise be empty MAY be maintained by creating a file within it named according to
-            local conventions, for example by making an empty <code>.keep</code> file.
-        </p>
+        <section id="content-directory">
```
Comment (zimeon) [inline draft/spec/index.html]: Missing a heading for the section

Comment (rosy1280) [reply draft/spec/index.html]: whoops!  added

File: draft/spec/index.html [hunk 2]
```diff
@@ -459,6 +459,7 @@ <h2>Version Directories</h2>
             sub-directory. Once created, the contents of a version directory are expected to be immutable.
         </p>
         <section id="content-directory">
+          <h3>Content Directory</h3>
```
Comment (zimeon) [inline draft/spec/index.html]: I think this should be `<h2>` ... respec uses this same level for all headings that show in the TOC

**Comment by rosy1280** [comment]
@ahankinson or @neilsjefferies if you approve, you can merge.

**Pull request merged**